pandas/core/generic.py

```diff
@@ -10397,43 +10397,50 @@ def _doc_parms(cls):
 Series([], dtype: bool)
 """
 
-_sum_examples = """\
+_shared_docs['stat_func_example'] = """\
 Examples
 --------
-``MultiIndex`` series example of monthly rainfall
+``MultiIndex`` series example.
```

Comment:
If this is the only example, I'd remove this sentence, as it's not adding much value. If we left it, MultiIndex would go between single backticks, and Series with capital S (I know it's from the original docstring, just letting you know).